src/apns_connection.erl

```diff
@@ -228,8 +235,8 @@ handle_info(reconnect, State) ->
       {ok, _} = timer:send_after(Sleep, reconnect),
       {noreply, State#{backoff => Backoff + 1}}
   end;
-handle_info(timeout, #{gun_connection := GunConn, client := Client} = State) ->
-  {ok, Timeout} = application:get_env(apns, timeout),
+handle_info(timeout, #{connection := Connection, gun_connection := GunConn, client := Client} = State) ->
```

Comment:
Line 238 is too long: handle_info(timeout, #{connection := Connection, gun_connection := GunConn, client := Client} = State) ->.